packages/ui/src/memberships/modals/UpdateMembershipModal/UpdateMembershipModal.tsx

```diff
@@ -46,9 +46,8 @@ export const UpdateMembershipModal = () => {
     )
   }
 
-  if (state.matches('success')) {
-    setActive(member)
-    return <UpdateMembershipSuccessModal onClose={hideModal} member={member} />
+  if (state.matches('success') && active != undefined) {
```

Comment:
Maybe `&& active` or `&& active !== undefined` is better?

Reply:
```suggestion
  if (active && state.matches('success')) {
```

Reply:
I have fixed the problem on the Jest test code and changed the code as following @traumschule.